core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -789,8 +790,19 @@ private void loadMetadata(LockedInodePath inodePath, LoadMetadataContext context
               LOG.debug("Failed to loadMetadata because file is not in ufs:"
                       + " inodePath={}, options={}.",
                   childURI, loadMetadataContext, e);
+            } catch (BlockInfoException | FileAlreadyCompletedException
+                | FileDoesNotExistException | InvalidFileSizeException
+                | IOException e) {
```

Comment:
Since you intend to catch the exceptions here, should you change the method signature?

Reply:
No, because invoking `loadFileMetadataInternal` and `loadDirectoryMetadata` throw these exceptions.